crates/matrix-sdk-crypto/src/store/mod.rs

```diff
@@ -913,6 +913,20 @@ impl From<&InboundGroupSession> for RoomKeyInfo {
     }
 }
 
+/// Information on a room key that has been withheld
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct RoomKeyWithheldInfo {
```

Comment:
Might make sense to use this in the store as well, maybe? But probably not worth to do a migration for that.

Reply:
Yeahhh. Maybe some other time.